### PR TITLE
feat: Refactor OG Image Generation 

### PR DESCRIPTION
**Pull request by neo773**
- Refactored Signing OG card to remove `getAssetBuffer`

- Added an API Route to get Signer info since `Prisma` can't be used in `edge` runtime and relative URL imports are not supported in `node`

- And moved all assets to `packages/public-assets` to remove all duplications and is symlinked back to their respective locations

image

/claim #637

### CODE REVIEW

**Comment by Mythie** [comment]
Hey there, it looks like you haven't accepted our contributor license agreement yet. In order for us to accept your pull request we ask that you please fill out the CLA:

https://documen.so/cla

**Comment by neo773** [comment]
> Hey there, it looks like you haven't accepted our contributor license agreement yet. In order for us to accept your pull request we ask that you please fill out the CLA:
> 
> https://documen.so/cla

Looks like it's stuck, I just accepted the CLA.

**Comment by Mythie** [comment]
Hey there, looks like some formatting changes came in with this PR! Ideally we'll be able to revert those so it's easier to review but I'll take a stab at seeing what's happened.

File: apps/web/src/app/(share)/share/[slug]/opengraph/route.tsx [hunk 1]
```diff
@@ -156,3 +171,5 @@ export async function GET(_request: Request, { params: { slug } }: SharePageOpen
     },
   );
 }
+
+export const runtime = 'edge';
```
Comment (Mythie) [inline apps/web/src/app/(share)/share/[slug]/opengraph/route.tsx:175]: Can this go top of file, I almost missed it 🤓

**Comment by Mythie** [comment]
So it seems like the issue was just ImageResponse being unhappy when used on a route-handler with the Node runtime? Is that the case or is there something more to this 🤔

**Comment by neo773** [comment]
> So it seems like the issue was just ImageResponse being unhappy when used on a route-handler with the Node runtime? Is that the case or is there something more to this 🤔

Yep that was it, I wasted like an hour trying to figure why it wouldn't work when the file path was right lol.

**Comment by neo773** [comment]
@Mythie 
Hi there, have you had a chance yet to review this?

**Comment by Mythie** [comment]
Hey sorry, sure haven’t! It’s ok my list for tomorrow if that’s okay?

**Review by Mythie — APPROVED** [review]

**Pull request merged**